src/reducers/notifications.ts

```diff
@@ -81,7 +82,7 @@ export const notificationsReducer = (
           ...topicState,
           existingIds: newFullIdsSet,
           fullNotifications: unshiftedNotifications,
-          hasMore: topicState?.hasMore || false,
+          hasMore: topicState?.hasMore ?? false,
```

Comment:
Again, `hasMore` is a boolean. I think we should change the `??` rule. In this specific instance it won't matter but a blanket change like this is bad